website/guide/programming.html

```diff
@@ -493,6 +493,32 @@ <h2 id="duktape-api">Duktape API</h2>
 duk_get_prop_string(ctx, -1, "my_state_variable");
 </pre>
 
+
+<h3>Hidden Symbol properties</h3>
```

Comment:
Maybe add an id to the section so it can be linked to:

```
<h3 id="hidden-symbol-properties">...</h3>
```

Reply:
Doh, will do.

Reply:
Since none of the other sections have an `id` attribute, would you prefer that I leave this single id change out and submit another PR adding `id` attributes to all the sections??

Reply:
Some of the sections in programming.html do have an id attribute, mostly when it's easy to anticipate they will be linked to:

```
<h1 id="programming">Programming model</h1>
<h2 id="duktape-api">Duktape API</h2>
<h2 id="error-handling">Error handling</h2>
<h2 id="normal-and-fatal-errors">Normal and fatal errors</h2>
```